contracts/cw-proposal-single/src/contract.rs

```diff
@@ -277,6 +282,10 @@ pub fn execute_execute(
         old_status.to_string(),
         prop.status.to_string(),
     )?;
+
+    // Update proposal's last updated timestamp.
+    prop.last_updated = env.block.time;
+
```

Comment:
```suggestion
```

Any particular reason to do this twice in `execute_execute` here?

Reply:
oh god, clearly I was writing this under heavy influence of brain fog